pkg/agent/controller/networkpolicy/reject.go

```diff
@@ -121,7 +121,7 @@ func (c *Controller) rejectRequest(pktIn *ofctrl.PacketIn) error {
 		if c.antreaProxyEnabled {
 			matches := pktIn.GetMatches()
 			if match := getMatchRegField(matches, openflow.ServiceEPStateField); match != nil {
-				if svcEpstate, err := getInfoInReg(match, openflow.ServiceEPStateField.GetRange().ToNXRange()); err != nil {
+				if svcEpstate, err := getInfoInReg(match, openflow.ServiceEPStateField.GetRange().ToNXRange()); err == nil {
```

Comment:
As you pointed out in the commit message, this is a bug fix. So it should be in its own PR. If someone just looks at the commit title ("Add containerName param to probe func"), they would have no idea that the commit includes a bug fix for the `Reject` implementation.

Reply:
Good one. Open PR #3010.